index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Title</title>
+</head>
+<body>
+<header class="outer-container">
+    <nav class="nav-bar">
+        <ul>
+            <li><a href="./case-studies.html">Case studies</a></li>
+            <li><a href="./services.html">Services</a></li>
+            <li><a href="./contact-us.html">Contact us</a></li>
+        </ul>
+    </nav>
+    <div class="hero-section">
+        <h1>Your website is an investment in the success of your business</h1>
+        <h2>We are here to help you succeed</h2>
+        <button type="button" class="button">Let's work together</button>
+    </div>
+    <img class="squiggle-line" src="./assets/squiggle_line_stripe.svg" alt="streepje">
+    <div class="schuine-lijn">Dit word de schuine lijn</div>
+</header>
+
+<main>
+<div class="intro">
+    <h3>Rekupe is a digital agency in Los Angeles.</h3>
+    <h3>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h3>
+    <h3>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h3>
+<p>Have a question? <a href="./contact-us.html" class="contact-us"> Click here&nbsp </a> to contact us</p>
+    </div>
+
+    <section>
+        <!--=================== Card 1 ==========================-->
+        <div class="card">
```

Comment:
Een < div > zegt semantisch eigenlijk vrij weinig, wanneer we wel belangrijke content willen aangeven in HTML kunnen we de < article > tag het beste daarvoor gebruiken.